### DISCUSSION TITLE
Upstreaming back into Yosys?

### DISCUSSION THREAD

**Discussion by mlyoung101** [povik/yosys-slang#45] [Q&A] [answered]
Hi!

This is a really cool project. I'm just wondering if there are plans to upstream the Slang frontend back into Yosys, or if it will stay as a plugin?

Thanks!

**Comment by povik** [comment, marked as answer]
Hi,

Yes, I would like for this to become a part of Yosys eventually. I have talked to other Yosys maintainers about it. So far it's a plugin to give me more freedom in how I develop it before it matures.
